x-pack/docs/en/security/authentication/security-domain.asciidoc

```diff
@@ -0,0 +1,97 @@
+[role="xpack"]
+[[security-domain]]
+=== Security domains
+
+beta::[]
+
+Security domains are a method of grouping <<realms,realms>> to enable resource
+sharing across different realms for the same user.
+
+==== How security domain works
+
+Users can create various resources in {es} that they own, access, and manage.
+For example, using the <<async-search,async search API>> to create a search
+request, <<security-api-create-api-key,creating an API key>>, or activating a
+<<user-profile,user profile>>.
+
+When a user creates a resource, {es} captures the user's `username` and realm
+information as part of the resource's metadata.
+When the user attempts to access the resource at a later time, {es} compares
+captured `username` and realm information against those from the accessing
+user. The access is allowed only when both of them match.
+That is, if {es} detects that a `username` from two different realms is
+attempting to access a resource, {es} assumes that these users are distinct and
+doesn't allow resources to be shared between them.
+
+However, there are cases where the same end-user can authenticate with
+different realms and need to share the same set of resources across the realms.
+For example, an <<ldap-realm,LDAP realm>> and a <<saml-realm,SAML realm>> can
+be backed by the same directory service. Additionally,
+<<configuring-authorization-delegation,authorization delegation>> allows one
+realm to delegate authorization to another realm. If both realms authenticate
+end-users with the same `username`, it is reasonable to treat these users to
+be the same user.
+
+With security domains, you can configure multiple realms to be under the same
+domain, users can authenticate with any of the realms and have access to the
+same set of resources regardless of which realm under the same domain they're
+authenticating from. For example, a single <<user-profile,user profile>> is
+associated to the user which enables user preferences, notifications, and other
+user data to be shared across domains. The same user can view results from an
```

Comment:
I found the "user which enables" to be hard to follow.
We could change it to "user, enabling ..."

Or something more like this:
```suggestion
authenticating from.
For example, two users who are linked by a single domain will
share a <<user-profile,user profile>>, and can share user
preferences, notifications, and other user data. 
The same user can view results from an
```

Reply:
I updated to `user, enabling ...` because your other version uses the term `link` and I am concerned this might be another cogonitive load for readers.